### DISCUSSION TITLE
Add Rubberwood Saplings to the Market

### DISCUSSION THREAD

**Discussion by TheBedrockMaster** [AllTheMods/All-the-mods-9-Sky#867] [Modpack Content]
Rubberwood from Thermal is basically unobtainable, I suggest adding it to the Market and/or from sieving moss unless this is intentional.

**Comment by llearch** [comment]
The trader has these saplings intermittently, if you're lucky (at least, in my playthrough). This may or may not be enough to meet requirements.

**Comment by TheBedrockMaster** [comment]
It is obtainable via sieving in dev now.
